File: Exercicios/Exerc_sem2/Python/Att_bonus2.py
# BONUS - DESAFIO 2

print("-----Calculadora-----")
print("1 - Soma")
print("2 - Subtração")
print("3 - Multiplicação")
print("4 - Divisão")
opcao = int(input("Escolha uma opção: "))

n1 = float(input("Primeiro valor: "))
n2 = float(input("Segundo valor: "))

if opcao == 1:
    print("Resultado: ", n1 + n2)
elif opcao == 2:
    print("Resultado: ", n1 - n2)
elif opcao == 3:
    print("Resultado: ", n1 * n2)
elif opcao == 4:
    if n1 != 0 and n2 != 0:
        print("Resultado: ", n1 / n2)
    else:
        print("Erro: Divisão por zero não permitida.")
else:
    print("Opção inválida!")

File: Exercicios/Exerc_sem2/C/Att_bonus2.c
#include <stdio.h>

void main() {
    int opcao;
    float n1, n2;

    printf("-----Calculadora-----");
    printf("\n 1 - Soma");
    printf("\n 2 - Subtraçao");
    printf("\n 3 - Multiplicacao");
    printf("\n 4 - Divisao");
    printf("\n Escolha uma opcao: ");
    scanf("%d", &opcao);

    printf("Primeiro valor: ");
    scanf("%f", &n1);
    printf("Segundo valor: ");
    scanf("%f", &n2);

    if (opcao == 1) {
        printf("Resultado: %.2f", n1 + n2);
    }
    else if (opcao == 2) {
        printf("Resultado: %.2f", n1 - n2);
    }
    else if (opcao == 3) {
        printf("Resultado: %.2f", n1 * n2);
    }
    else if (opcao == 4) {
        if (n2 != 0) {
            printf("Resultado: %.2f", n1 / n2);
        } else {
            printf("Erro: Divisao por zero nao permitida.");
        }
    }
    else {
        printf("Opcao invalida!");
    }
}
